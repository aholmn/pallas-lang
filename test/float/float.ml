let () =
  Interpreter.interpreter "../../examples/float"
