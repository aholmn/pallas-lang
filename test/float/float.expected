1
2
3
4
5
6
7
8
9
1.12345
2.12345
3.12345
4.12345
5.12345
6.12345
7.12345
8.12345
9.12345
